### PR TITLE
fixes for rhel7 at SPI

### PR DESCRIPTION
**Pull request by cmstein**
Build changes to compile on rhel7 for SPI

### CODE REVIEW

File: site/spi/Makefile-bits-arnold [hunk 1]
```diff
@@ -161,6 +161,127 @@ ifeq ($(SP_OS), spinux1)
 
 endif  # endif spinux1
 
+## Rhel7 (current)
+ifeq ($(SP_OS), rhel7)
+    platform=rhel7
+    SPCOMP2_COMPILER=gcc48m64
+    INSTALL_SPCOMP2_LOCATION = /shots/spi/home/lib/SpComp2
+    # At SPI, we have two "flavors" of spinux.  One is based on Foresight, which
```
Comment (fpsunflower) [inline site/spi/Makefile-bits-arnold]: Is this comment still relevant to the new OS?

File: site/spi/Makefile-bits-arnold [hunk 2]
```diff
@@ -161,6 +161,127 @@ ifeq ($(SP_OS), spinux1)
...
+    ifeq (${COMPILER},clang)
+        MY_CMAKE_FLAGS += -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
+    endif
+    ifeq (${COMPILER},gcc)
+        MY_CMAKE_FLAGS += -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++
+    endif
+
+    # A variety of tags can be used to try specific versions of gcc or
+    # clang from the site-specific places we have installed them.
+    ifeq (${COMPILER}, clang341)
```
Comment (lgritz) [inline site/spi/Makefile-bits-arnold]: Any reason to keep clang 3.4.1?

File: site/spi/Makefile-bits-arnold [hunk 3]
```diff
@@ -161,6 +161,127 @@ ifeq ($(SP_OS), spinux1)
...
+        MY_CMAKE_FLAGS += \
+           -DCMAKE_C_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.1/bin/clang \
+           -DCMAKE_CXX_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.1/bin/clang++
+    endif
+    ifeq (${COMPILER}, clang342)
+        MY_CMAKE_FLAGS += \
+           -DCMAKE_C_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.2/bin/clang \
+           -DCMAKE_CXX_COMPILER=/shots/spi/home/lib/arnold/spinux1/llvm_3.4.2/bin/clang++
+    endif
+    ifeq (${COMPILER}, gcc463)
```
Comment (lgritz) [inline site/spi/Makefile-bits-arnold]: Just picking this one as an example... but are there any of these gcc variants that we know we'll never need?

**Comment by lgritz** [comment]
LGTM. I was just musing about whether any of the accumulated cruft can be trimmed.
Merge when you're ready, there's no way this can break anybody else.
